frontend/src/citizen-frontend/calendar/CalendarListView.tsx

```diff
@@ -18,7 +18,7 @@ export interface Props {
   selectDate: (date: LocalDate) => void
 }
 
-export default React.memo(function CalendarListView({
+export default React.memo(function CalendarGridView({
```

Comment:
You have the two function names mixed up.

Reply:
oops!